### PR TITLE
#797 Conditionally disable "Allow All" input on IPVS KUBE-ROUTER-SERVICES

### PR DESCRIPTION
**Pull request by DandyDeveloper**
See #797 for more details. 

fix#797
fix#682

This is to support people wanting to prevent all incoming connections from being accepted. 

In our case, we selectively whitelist items, so we would rather have this specific rule disabled. 

Can someone confirm if there's a better naming convention here for these types of flags?

### CODE REVIEW

**Comment by DandyDeveloper** [comment]
I have tried running tests against this, but it looks like the tests in master are currently broken / not working. 

Is anyone working to fix this?

**Comment by DandyDeveloper** [comment]
This needs more testing. This breaks some networking.

File: docs/user-guide.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Usage of kube-router:
       --ipvs-graceful-period duration                 The graceful period before removing destinations from IPVS services (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 30s)
       --ipvs-graceful-termination                     Enables the experimental IPVS graceful terminaton capability
       --ipvs-sync-period duration                     The delay between ipvs config synchronizations (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 5m0s)
+      --ipvs-allow-all-input                          Enables rule to accept all incoming traffic on the node. (default true).
```
Comment (andrewsykim) [inline docs/user-guide.md]: prefer --ipvs-deny-all which defaults to false

Comment (DandyDeveloper) [reply docs/user-guide.md]: @andrewsykim We may be better off just not applying the chain based on my testing. I am spending a little more time on this just to make sure it does exactly what's expected and has no adverse negative impact.

Comment (andrewsykim) [reply docs/user-guide.md]: thanks, appreciate the thoroughness

**Pull request closed without merging**